client/server.go

```diff
@@ -334,6 +341,23 @@ func (server *CqlServer) BindAndInit(
 	}
 }
 
+type Response struct {
```

Comment:
Does this new type need to be exported? It seems it's rather an internal type.

Reply:
nope, renamed it